docs/architecture/modernize-desktop/windows-migration.md

```diff
@@ -0,0 +1,527 @@
+redirected writes to AppData or the registry that were captured during the
+process.
+
+For details about how a packaged application handles installation, file access,
+registry, and uninstallation read
+<https://docs.microsoft.com/en-us/windows/msix/desktop/desktop-to-uwp-behind-the-scenes>
+
+You can get a complete list of things to check on
+<https://docs.microsoft.com/en-us/windows/msix/desktop/desktop-to-uwp-prepare>
+
+## How to add WinRT APIs to your desktop project
+
+In this section, you can find a walkthrough about how to integrate Toast
+Notifications on an existing WPF application. Although it is simple from the
+code perspective, it helps illustrate the whole process. Notifications are one
+of the many available WinRT APIs available that you can use in .NET app. In this
+case, the API requires a Package Identity. This process is more straightforward
+if the APIs don't require Package Identity.
+
+Let's take an existing WPF sample app that reads files and shows its contents on
+the screen. The goal will be to display a Toast Notification when the
+application starts.
+
+![Sample application](media\windows-migration\sample-application.png)
+
+First, you should check in the below link whether the Windows 10 API that you
+will use requires a Package Identity:
+
+<https://docs.microsoft.com/en-us/windows/apps/desktop/modernize/desktop-to-uwp-supported-api>
+
+Our sample will use the Windows.UI.Notifications API that requires a packaged
+identity:
+
+![Notification class in Microsoft documentation](media\windows-migration\notification-class-documentation.png)
+
+To enable access to the Windows Runtime API, you just need to add a reference to
+the Microsoft.Windows.SDK.Contracts NuGet package and this will do the magic
+behind the scenes (see details in this link:
+<https://blogs.windows.com/windowsdeveloper/2019/04/30/calling-windows-10-apis-from-a-desktop-application-just-got-easier/\#OQKKZIOZu4p7lC0q.97>)
+
+You are now prepared to start adding some code.
+
+Create a ShowToastNotification method that will be called on
+application startup. It just builds a toast notification from an XML pattern:
+
+```csharp
+private void ShowNotification(string title, string content, string image)
+{
+    string xmlString = $@"<toast><visual><binding template = 'ToastGeneric'><text>{title}</text><text>{content}</text><image src=>'{image}'</image></binding></visual></toast>";
+    XmlDocument toastXml = new XmlDocument();
+    toastXml.LoadXml(xmlString);
+    ToastNotification toast = new ToastNotification(toastXml);
+    ToastNotificationManager.CreateToastNotifier().Show(toast);
+}
+```
+
+Although the project builds, there are errors because the Notifications API
+requires a Package Identity, and we didn't provide it. Adding a Windows
+Packaging Project in the Solution will fix it:
+
+![Adding a Windows Packaging Project](media\windows-migration\adding-packaging-project.png)
+
```

Comment:
don't use ing for the file names. so instead of adding, add. same for the next image, instead of selecting, select, etc. lots of images on this PR with the same problem.